kedro/framework/cli/utils.py

```diff
@@ -422,9 +423,12 @@ def _reformat_load_versions(  # pylint: disable=unused-argument
     """Reformat data structure from tuple to dictionary for `load-version`, e.g.:
     ('dataset1:time1', 'dataset2:time2') -> {"dataset1": "time1", "dataset2": "time2"}.
     """
-    _deprecate_options(ctx, param, value)
+    if param.name != "load_versions":
```

Comment:
```suggestion
    if param.name == "load_version":
```
Feels a bit easier to understand if we condition based on what we _are_ deprecating here rather than what we're not.